code/modules/vehicles/tank.dm

```diff
@@ -319,4 +319,29 @@
 	if(gunner)
 		log_combat(gunner, null, "[src] took [damage] [type] damage [ismob(attacker) ? "from [key_name(attacker)]" : ""].")
 	if(driver)
-		log_combat(driver, null, "[src] took [damage] [type] damage [ismob(attacker) ? "from [key_name(attacker)]" : ""].")
\ No newline at end of file
+		log_combat(driver, null, "[src] took [damage] [type] damage [ismob(attacker) ? "from [key_name(attacker)]" : ""].")
+
+
+/obj/vehicle/multitile/root/cm_armored/proc/click_action(A, mob/user, params)
+	var/list/mods = params2list(params)
+	if(istype(A, /obj/screen) || A == src || mods["middle"] || mods["shift"] || mods["alt"])
```

Comment:
This should as well be handled by the screen Click redefinition, no?

Reply:
As I said relic /tg/ code handles mechs in the same way. Larger refactor needed